internal/alertmanager/currentalert/process.go

```diff
@@ -107,6 +116,9 @@ func (e *examiner) processAlert(ds disableScaling) (bool, error) {
 
 		return true, nil
 	case "upScale":
+		if err := checkLabelExists(e.Alert.Labels, "kafka_cr"); err != nil {
```

Comment:
I'd introduce types for alerts and define a validation function which could be invoked after filtering.